### DISCUSSION TITLE
Is there any typo in stream protocol document?

### DISCUSSION THREAD

**Discussion by bangjiehan** [rabbitmq/rabbitmq-server#10732] [Questions]
I just got some confused about frame-structure section.
[https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbitmq_stream/docs/PROTOCOL.adoc#frame-structure](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbitmq_stream/docs/PROTOCOL.adoc#frame-structure)

**Comment by acogoluegnes** [comment]
Please be more explicit. What is it you are confused about?

**Reply by bangjiehan** [reply]
```
Request => Key Version (CorrelationId) Content <- Here is Content
  Key => uint16
  Version => uint16
  CorrelationId => uint32
  Command => bytes // see command details below <- Here is Command
```
